static/assets/contribute-templates/how-to-template.md

```diff
@@ -1,36 +1,631 @@
 
-1. Open ...
+It's important to prepare both runtimes and push them consecutively. First prepare the normal runtime you are trying to upgrade with and save it aside for use in step 10. Now, let's create this minimalistic runtime. It must be smaller than your present block length. In our case, the block length was 65KB. All of the following changes occur in `runtime/src/lib.rs`.
 
-## Examples [required - link to at least one example]
 
-Replace this paragraph with an introductory sentence followed by a link to at least one working example that corresponds with the information covered in this guide.
+1. Remove any and all custom code you added to your runtime. This means removing the implementation from `construct_runtime!`, all associated `parameter_types!`, and all implementations for runtime.
 
-## Related resources [optional - add links as needed]
+2. Comment out some unneeded configs and functions. Things like `native_version()`, `frame_benchmarking`, and `pallet_transaction_payment_rpc_runtime_api` can all be removed. 
+
+   ```rust
+   // #[cfg(feature = "std")]
+   // pub fn native_version() -> NativeVersion {
+   // 	NativeVersion {
+   // 		runtime_version: VERSION,
+   // 		can_author_with: Default::default(),
+   // 	}
+   // }
+
+3. Remove all unneeded `parameter_types!`. 
+
+4. This is the hard part, remove all unneeded runtime pallet traits! This is what our `construct_runtime!` looked like when we were finished with it:
+```rust
+construct_runtime!(
+	pub enum Runtime where
+		Block = Block,
+		NodeBlock = opaque::Block,
+		UncheckedExtrinsic = UncheckedExtrinsic
+	{
+		System: frame_system::{Call, Event<T>},
+		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet},
+		Timestamp: pallet_timestamp::{Call, Inherent},
+		Aura: pallet_aura::{Pallet},
+		Grandpa: pallet_grandpa::{Call, Event},
+		Balances: pallet_balances::{Event<T>},
+		TransactionPayment: pallet_transaction_payment::{Pallet},
+		Sudo: pallet_sudo::{Call, Event<T>}
+	}
+);
+```
+  This is the bare minimum needed for our blockchain to work, note how we removed all `Config<T>` and `Event<T>` where they weren't being used. 
+
+5. Remove all unnecessary `signedExtensions`. 
+  ```rust
+  pub type SignedExtra = (
+    frame_system::CheckSpecVersion<Runtime>,
+    //frame_system::CheckTxVersion<Runtime>,
+    // frame_system::CheckGenesis<Runtime>,
+    // frame_system::CheckEra<Runtime>,
+    // frame_system::CheckNonce<Runtime>,
+    // frame_system::CheckWeight<Runtime>,
+    // pallet_transaction_payment::ChargeTransactionPayment<Runtime>
```

Comment:
might be worth mentioning that removing some of these makes your chain be easily attackable. I would personally NOT remove these in your scenario. You need some of these to keep the chain safe. 

For example, not checking the Nonce is basically suicide if any pallet is active to end-users. 

Or not checking the Genesis hash makes you vulnerable to long-range attacks.

Reply:
@kianenigma Agree with you it's rather dangerous. However in the article we do mention that this minimalist runtime should be followed **immediately** by another upgrade that will bring back your original code. I will add an update to the article to mention your points.